test/visual-regression/alert-group.spec.ts

```diff
@@ -11,21 +11,27 @@ test("alert group default", async ({ page }) => {
   await page.getByTestId("fudis-button-2").click();
 
   // Close two previous alerts
-  await page.getByTestId("fudis-alert-1-button").click();
   await page.getByTestId("fudis-alert-2-button").click();
-  await expect(page).toHaveScreenshot("2-add-and-remove.png");
+  await page.getByTestId("fudis-alert-4-button").click();
+  await expect(page.getByText("Nothing special here.")).not.toBeVisible();
+  await expect(page.getByText("Yippee Ki-Yay! You were successful!")).not.toBeVisible();
 
   // Open dialog
-  await page.getByTestId("fudis-button-7").click();
-  await page.getByTestId("fudis-alert-3-button").click(); // Dismiss one Alert
+  await page.getByTestId("fudis-alert-6-button").click(); // Dismiss one Alert
+  await page.getByTestId("fudis-button-3").click(); // Add success Alert
+  await page.getByTestId("fudis-button-7").click(); // Open dialog
```

Comment:
I would suggest adding expects after these button clicks; check that new alert is added and dialog is opened.

I would also add an expect after dialog is closed with Escape keyboard press.